parts/k8s/kubernetesinstalls.sh

```diff
@@ -86,11 +113,7 @@ installSGXDrivers() {
 
 installContainerRuntime() {
     if [[ "$CONTAINER_RUNTIME" == "docker" ]]; then
-        if [[ "$DOCKER_ENGINE_REPO" != "" ]]; then
```

Comment:
I think we still want to allow a user to specify docker-engine

Reply:
https://github.com/Azure/aks-engine/blob/master/pkg/engine/params_k8s.go#L240 is what you want to change